src/ProjectFileTools.MSBuild/ProjectFileTools.MSBuild.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
+  <PropertyGroup>
+    <TargetFrameworks>net46</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net46'">
+    <PackageReference Include="Microsoft.Build" Version="15.3.0-preview-000117-01" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Language.Xml" Version="1.1.11" />
+    <Reference Update="**\Microsoft.Language.Xml.dll" CopyLocal="true" />
```

Comment:
+1 if the file was in the output before adding this, only the vsixmanifest bit should be required to get the dll included in the vsix